site/docs/4.5/utilities/interactions.md

```diff
@@ -16,4 +16,6 @@ Change how the content is selected when the user interacts with it.
 {% endcapture %}
 {% include example.html content=example %}
 
+**Note: `user-select-all` is not supported on Internet Explorer or Edge Legacy.**
```

Comment:
I'd move this up to be part of Line 10, and make this edit:

> Note that Internet Explorer and Legacy Edge have no support for the `all` value for `user-select`, and as such, **`.user-select-all` is not supported by either browser.**

Reply:
@mdo can you make a PR please? ❤